addons/mail/wizard/mail_compose_message.py

```diff
@@ -722,18 +738,12 @@ def _action_schedule_message(self):
             if not post_values['scheduled_date']:
                 raise UserError(_("A scheduled date is needed to schedule a message"))
             create_values.append({
-                'attachment_ids': post_values.pop('attachment_ids'),
-                'author_id': post_values.pop('author_id'),
-                'body': post_values.pop('body'),
+                **wizard._prepare_schedule_message_post_values(post_values),
```

Comment:
Would prepare all values, (including those wizard dependant). That way caller / overrider (does that word mean anything ?) has access to all values, not only part of it.